<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDCMotorFrame</class>
 <widget class="QFrame" name="OutputDCMotorFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="labelState">
       <property name="text">
        <string>Forward</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="buttonRev">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rev</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="buttonFwd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fwd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="buttonBrake">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Brake</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QSlider" name="slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>DC Motor 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="4">
      <widget class="QPushButton" name="buttonOverride">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Override</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
